src/wwwroot/app/views/home/home.controller.js

```diff
@@ -35,10 +44,16 @@ export default function VacanciesController(
          vm.vacancies = response.vacancies;
       }).catch(_onError);
    };
+
    UserService.getUsers().then(users => set(vm, 'responsibles', users));
 
    ThesaurusService.getThesaurusTopicsGroup(LIST_OF_THESAURUS).then(topics => set(vm, 'thesaurus', topics));
 
+   NoteService.getNotesByUser().then((notes) => {
```

Comment:
may be the initialization of thesauruses, users and notes to relocate into _init function?